src/js/components/covid19/awardSpendingAgency/AwardSpendingAgency.jsx

```diff
@@ -70,8 +72,10 @@ const AwardSpendingAgency = () => {
                     def_codes: defCodes.map((defc) => defc.code)
                 }
             };
-            if (awardType.internal !== 'all') {
-                // Endpoint defaults to all financial assistance types if award_type_codes is not provided
+            if (awardType.internal === 'all') {
+                const allAwardTypeGroups = [];
+                params.filter.award_type_codes = allAwardTypeGroups.concat(...Object.values(awardTypeGroups).map((awardTypeGroup) => awardTypeGroup));
```

Comment:
don't think we need the `.map` here

```suggestion
                params.filter.award_type_codes = allAwardTypeGroups.concat(Object.values(awardTypeGroups));
```